Gemfile.lock

```diff
@@ -432,6 +432,8 @@ GEM
     js-routes (2.2.4)
       railties (>= 4)
     json (2.6.1)
+    json-schema (2.8.1)
```

Comment:
I wasn't going to add this for the feed variant stuff, but now that we have it I might be adding some schema definitions.